modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSMessageStoreImpl.java

```diff
@@ -700,23 +742,24 @@ public void updateMetadataInformation(String currentQueueName, List<AndesMessage
      * @param queueName         queue to be assigned
      * @throws SQLException
      */
-    private void addMetadataToBatch(PreparedStatement preparedStatement,
-                                    AndesMessageMetadata metadata,
-                                    final String queueName) throws SQLException {
+    private void addMetadataToBatch(PreparedStatement preparedStatement, AndesMessageMetadata metadata,
+                                    final String queueName) throws AndesException {
 
-        Context metaAdditionToBatchContext = MetricManager.timer(Level.INFO, MetricsConstants.ADD_META_DATA_TO_BATCH).start();
+        Context metaAdditionToBatchContext = MetricManager.timer(Level.INFO, MetricsConstants.ADD_META_DATA_TO_BATCH)
```

Comment:
@AnujaLK 
metaAdditionToBatchContext and contextWrite are used for the same purpose. Why do we need two? Can't we use just one?